sdk/src/types/block/output/account.rs

```diff
@@ -33,6 +33,30 @@ use crate::types::{
     ValidationParams,
 };
 
+impl_id!(pub AccountId, 32, "Unique identifier of an account, which is the BLAKE2b-256 hash of the Output ID that created it.");
+
+#[cfg(feature = "serde")]
+string_serde_impl!(AccountId);
+
+impl From<&OutputId> for AccountId {
+    fn from(output_id: &OutputId) -> Self {
+        Self::from(output_id.hash())
+    }
+}
+
+impl AccountId {
+    ///
```

Comment:
docs?

Reply:
Outside of the scope of this PR, will do later